test/TensorFlowRuntime/tensor_autodiff_runtime.swift

```diff
@@ -4,7 +4,9 @@
 // VJP-based AD. See SR-9638.
 //
 // REQUIRES: executable_test
-// REQUIRES: swift_test_mode_optimize
+//
+// FIXME(TF-199): Indirect passing differentiation crashes with `-O`.
```

Comment:
We shouldn't disable `-O` for existing tests. Could you create a new file for testing indirect passing?

Reply:
Very good point.
Created new `tensor_autodiff_indirect.swift` test in https://github.com/apple/swift/pull/22544/commits/ad7436a8e38562bbf4aa6cf6ebe6918c829c5331. `-O` is re-supported for `tensor_autodiff_runtime.swift`.